server/webapp/WEB-INF/rails/app/helpers/agents_helper.rb

```diff
@@ -18,12 +18,17 @@ module AgentsHelper
 
   AgentStatus = com.thoughtworks.go.domain.AgentStatus
 
-  def agent_selector uuid, selector_name, selected_uuids
-    "<td class='selector'>#{agent_selector_without_cell(uuid, selector_name, selected_uuids)}</td>".html_safe
+  def agent_selector uuid, selector_name, selected_uuids, is_remote_agent=false
```

Comment:
I was confused when I saw `is_remote_agent`. Should we consider renaming to something like `is_associated_through_config_repo`.

 I guess the reason for using remote is because of the method `MergeEnvironmentConfig#containsAgentRemotely`

Reply:
Yeah, there are many usages of the `remote` and `local` terminology in the code. But I also find it confusing. So I've changed the name as you suggested.